msteams-platform/includes/custom-engine-agent.md

```diff
@@ -0,0 +1,58 @@
+> [!NOTE]
+>
+> ## Add support for Microsoft 365 Copilot chat in your custom engine agent
+   >
+   > If you want your custom engine agent to support Microsoft 365 Copilot chat, you must [install a prerelease version of Teams Toolkit](../toolkit/install-Teams-Toolkit.md#install-a-prerelease-version). After you install the prerelease version, follow these steps:
+   >
+   > 1. Add the environment variable, `TEAMSFX_CEA_ENABLED=true`.
```

Comment:
Is the flag still required for developers to enable at the release? If so, I think this needs explicit instructions on how.

Reply:
It is an environmental variable in the system and discussed the same with developer as they all aware of this.